code/controllers/subsystem/rogue/devotion.dm

```diff
@@ -8,8 +8,8 @@
 #define CLERIC_REQ_0 0
 #define CLERIC_REQ_1 100
 #define CLERIC_REQ_2 250
-#define CLERIC_REQ_3 500
-#define CLERIC_REQ_4 750
+#define CLERIC_REQ_3 750
+#define CLERIC_REQ_4 1000
```

Comment:
what about T3 and T4 costs?

Reply:
Oh shit you right I thought they were 750 and 1000